docs/connectors/README.md

```diff
@@ -10,36 +10,32 @@ Welcome to the Connectors Early Preview program!
 The Connector plugin is pre-installed in commercial versions of EventStoreDB but not enabled by default. 
 :::
 
-Connectors make it easy to integrate EventStoreDB data into other systems.
+Connectors simplify the integration of EventStoreDB data into other systems by reducing the work required to process data from EventStoreDB instances.  Connectors also reduce the work required to create data pipelines when implementing complex use cases.
 
-Each connector runs on the server-side and uses a catch-up subscription to receive events, filter or transform them, and push them to an external system via a [sink](https://en.wikipedia.org/wiki/Sink_(computing)).
+Each connector runs on the server-side and uses a catch-up subscription to:
+1. Receive events
+2. Filter (or transform) events
+3. Push events to an external system via a [sink](https://en.wikipedia.org/wiki/Sink_(computing))
 
 ![Connectors Anatomy](./images/connector-anatomy.png)
 
-This reduces the amount of work needed to process data from EventStoreDB instances and makes it easy to create data pipelines to implement complex use cases.
-
 In this preview, there are two sinks:
 
 - [Console Sink](./sinks.md#console-sink) for experimentation
 - [HTTP Sink](./sinks.md#http-sink) for sending events to an HTTP endpoint
 
 ## Motivation
 
-Currently, a pain point that users experience is that, on one hand, they have a convenient EventStoreDB cloud service, on the other hand, they have a convenient downstream database or processing engine, but there's nothing in between. As a result, users need to host and maintain their own
-solution in their own infrastructure for subscribing to EventStoreDB and
-sending the events to a downstream service. This solution in the middle often needs to be highly available and needs to manage its own checkpoints: this quickly becomes cumbersome.
-
-EventStore Connectors remove the need for users to develop, host and maintain such a solution.
-
-As an example, a developers want to send events to a lambda function, which projects events to an RDS instance.
+Users can experience a pain point when subscribing to EventStoreDB and
```

Comment:
Part of the last sentence "within the current infrastructure" is unclear to me. I can suggest the following instead:

Users may encounter challenges when subscribing to EventStoreDB for forwarding events to downstream services. This process often requires implementing, hosting, and maintaining a highly available solution that manages its own checkpoints. Managing such a  custom solution can quickly become cumbersome.